File: result.php




<?PHP

    $idval = intval($_GET['id']);
	$value=$_GET['val'];
    
	// mysql server
 	$dbserver = 'localhost';
	$dbuser = 'testuser';
	$dbpass = 'test123';
	$dbname ='testdb';
	
	
	
	$con = mysqli_connect($dbserver,$dbuser,$dbpass,$dbname);
	
	$sql = "select * from result where id=$idval";
	$result = mysqli_query($con,$sql) or die(mysqli_error(). "<hr/>Line: ". __LINE__ ."<br/>$sql");
	 $rs = mysqli_fetch_array($result);
	 if ($value=='imagename'){
	 	echo  $rs[1];
	 }
	 elseif ($value=='username'){
		 echo $rs[2];
	 }
	 elseif ($value=='tweet'){
		 echo $rs[3];
	 }
	 else{
		 echo "invalid";
	 }
	 mysqli_free_result($result); 
	mysqli_close($con);
	//echo $idval;
	//echo $value;
	
?>

File: result.py
#!/usr/bin/python3

import pymysql

def add_result(idval,imgName,username,tweet):
   # Open database connection
   db = pymysql.connect("localhost","testuser","test123","testdb" )

   # prepare a cursor object using cursor() method
   cursor = db.cursor()




   # Create table as per requirement
   sql = "INSERT INTO RESULT(ID , \
           IMAGENAME, USERNAME, 	TWEET) \
      VALUES ('%d', '%s', '%s', '%s')" % \
      (idval, imgName,username,tweet)

   try:
      # Execute the SQL command
      cursor.execute(sql)
      # Commit your changes in the database
      db.commit()
   except:
      # Rollback in case there is any error
      db.rollback()

   # disconnect from server
   db.close()
#add_result(2,'hi.jpg','THacksters','hello everyone')
